linq4j/src/main/java/org/apache/calcite/linq4j/LookupImpl.java

```diff
@@ -88,8 +88,7 @@ public boolean containsKey(Object key) {
   public boolean containsValue(Object value) {
     @SuppressWarnings("unchecked")
     List<V> list = (List<V>) value;
-    Enumerable<V> enumerable = Linq4j.asEnumerable(list);
-    return map.containsValue(enumerable);
+    return map.containsValue(list);
```

Comment:
this method looks to be broken - it will always return false. `Linq4j.asEnumerable(List)` returns a new object that does not override `hashCode`. if we're going to fix it, let's add a test.

Reply:
added test class LookupImplTest